tests/Unit/ClientTest.php

```diff
@@ -879,14 +899,14 @@ public function testUserReportTypes(): void
     {
         $this->guzzle_handler->onUriRequested(
             $this->settings->getBaseUri() . 'user/report_types?' . \http_build_query([
-                '_can_generate' => 'true',
                 '_content'      => 'true',
                 '_query'        => '_all',
                 '_size'         => 20,
                 '_offset'       => 0,
                 '_page'         => 1,
                 '_sort'         => '-created_at',
                 '_calc_total'   => 'true',
+                '_can_generate' => 'true',
```

Comment:
@tarampampam 

Why the test is edited?

This is the specifics of refactored query building. Now all common GET-parameters in the URL go first, and unique for the request at the end.